src/UserActivity.js

```diff
@@ -0,0 +1,39 @@
+class UserActivity {
+  constructor(userActivity){
+    this.userActivity = userActivity
+  }
+  calculateMiles(date){
+    let milesPerDate = this.userActivity.find
+
+    //we need to access this data from two different databases
```

Comment:
Maybe we can add a property that acts as the stride length to get this calculation?